tensionmap.py

```diff
@@ -155,6 +155,9 @@ def tm_update(obj, context):
     if obj.data.tm_enable_vertex_colors:
         vertex_colors = [0.0] * (number_of_tm_channels * num_vertices)
 
+    #lambda for clamping between min and max
```

Comment:
Don't forget the space after the `#`
I don't know what IDE you use, but if you want to have the same auto formatting as me, I use Visual Studio Code with the `Python` extension, and the `autopep8` formater

```json
"python.formatting.provider": "autopep8"
```

If you use VS Code, I would also recommand `MagicPython` for development.

Reply:
Fixed.
I actually edit it in Blender, since it's quicker to just save/reload/check performance.
I always wanted to just switch to PyCharm, but I have pushed it off until now...

Reply:
I personally have Blender and the User Prefs always opened, then when I update the code in VSCode, I delete the addon, then click install again, and I am presented with always the same folder that only contains the addon, then I enable it and it's done.
So it's relatively fast.